src/backend/app.js

```diff
@@ -1,35 +1,168 @@
-const express = require("express");
-const app = express();
-const router = express.Router();
-const path = require("path");
+const express = require('express')
+const app = express()
+const router = express.Router()
+const path = require('path')
 
-const mealsRouter = require("./api/meals");
-const buildPath = path.join(__dirname, "../../dist");
-const port = process.env.PORT || 3000;
-const cors = require("cors");
+const mealsRouter = require('./api/meals')
+const buildPath = path.join(__dirname, '../../dist')
+const port = process.env.PORT || 3000
+const cors = require('cors')
 
 // For week4 no need to look into this!
 // Serve the built client html
-app.use(express.static(buildPath));
+app.use(express.static(buildPath))
 
 // Parse URL-encoded bodies (as sent by HTML forms)
-app.use(express.urlencoded({ extended: true }));
+app.use(express.urlencoded({ extended: true }))
 // Parse JSON bodies (as sent by API clients)
-app.use(express.json());
+app.use(express.json())
 
-app.use(cors());
+app.use(cors())
 
-router.use("/meals", mealsRouter);
+//localhost:5000/api/meals
+router.use('/meals', mealsRouter)
+
+//--Homework week1--
+
+//I like this solution, but Ahmed recommended another one
```

Comment:
It is cool that you found the `params`!
However, usually we use them when an endpoint needs an id or the like. E.g `/products/:productId`.
Here is an [article](https://betterprogramming.pub/10-best-practices-for-naming-rest-api-endpoints-442ae592a3a0) that explains it quite well.